new.downstream.stages

```diff
@@ -0,0 +1,14 @@
+stage('New Downstream Build') {
+    final REPOSITORY_LIST_FILE = "./script/repository-list.txt"
+    final SETTINGS_XML_ID = "771ff52a-a8b4-40e6-9b22-d54c7314aa1e"
+    final PROPERTIES_FILE_ID = "387b2726-55c0-4fcf-b5c0-ee7f4174d58a"
+    configFileProvider([configFile(fileId: SETTINGS_XML_ID, variable: 'MAVEN_SETTINGS_XML_DOWNSTREAM')]) {
+        println "Reading file ${REPOSITORY_LIST_FILE}"
+        def file = readFile REPOSITORY_LIST_FILE
+        def projectCollection = file.readLines()
+        def gitURL = env.ghprbAuthorRepoGitUrl ?: env.GIT_URL
+        def project = util.getProjectGroupName(util.getProject(gitURL))[1]
+        projectCollection.removeAll(['kie-wb-distributions', 'droolsjbpm-tools', 'kie-docs'])
```

Comment:
@Ginxo can you please remove 'kie-wb-distributions' here - since this project should build

Reply:
done